modules/alloy/CHANGELOG.md

```diff
@@ -28,6 +28,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Fixed a number of positioning bugs when using an "inset" layout, especially any that renders at the bottom.
 - `LayoutInset` bubble classes were applied incorrectly, causing the bubble arrows to show on the opposite side.
 - `MakeshiftAnchorSpec` did not extend `HasLayoutAnchorSpec` which meant layouts couldn't be used for makeshift anchors.
+- Disabled nested menu items could still be opened.
```

Comment:
This needs to be a little different to the TinyMCE changelog entry... so it probably needs to be something more like this:
```suggestion
- `TieredMenu` child menus were incorrectly opened when the parent menu item was disabled.
```